i18n/en-us.js

```diff
@@ -278,26 +278,49 @@ export default {
       ],
     },
     participant: {
-      name: '',
```

Comment:
Do you think we need to capitalize the first letter?

Reply:
Yes